x-pack/plugins/cases/server/services/attachments/index.ts

```diff
@@ -207,7 +173,7 @@ export class AttachmentService {
         );
 
       const attachment =
-        await this.context.unsecuredSavedObjectsClient.create<AttachmentAttributesWithoutRefs>(
+        await this.context.unsecuredSavedObjectsClient.create<AttachmentPersistedAttributes>(
```

Comment:
What about `bulkDelete`, `update`, and `bulkUpdate`?

Reply:
🤦‍♂️ don't know why I missed all that.

Reply:
Looks like `bulkDelete` doesn't take a type. I'll update the others though.